mlflow_elasticsearchstore/elasticsearch_store.py

```diff
@@ -233,24 +243,92 @@ def list_all_columns(self, experiment_id: str, run_view_type: str) -> Columns:
         tags = [t.key for t in response.aggregations.tags.tags_keys.buckets]
         return Columns(metrics=metrics, params=params, tags=tags)
 
+    def _build_elasticsearch_query(self, parsed_filters: List[dict], s: Search) -> Search:
+        type_dict = {"metric": "latest_metrics", "parameter": "params", "tag": "tags"}
+        filter_key = {
+            ">": ["range", "must"],
+            ">=": ["range", "must"],
+            "=": ["term", "must"],
+            "!=": ["term", "must_not"],
+            "<=": ["range", "must"],
+            "<": ["range", "must"],
+            "LIKE": ["wildcard", "must"],
+            "ILIKE": ["wildcard", "must_not"],
+        }
+        for search_filter in parsed_filters:
+            key_type = search_filter.get('type')
+            key_name = search_filter.get('key')
+            value = search_filter.get('value')
+            comparator = search_filter.get('comparator').upper()
+            filter_ops = {
+                ">": {'gt': value},
+                ">=": {'gte': value},
+                "=": value,
+                "!=": value,
+                "<=": {'lte': value},
+                "<": {'lt': value}
+            }
+            if comparator == "LIKE" or comparator == "ILIKE":
```

Comment:
you can use comparator in ["LIKE", "ILIKE"] here

Reply:
The value of this depends on what is value, not sure I can do this